### PR TITLE
updated information

### PR DESCRIPTION
**Pull request by lpbarber**
updated information, pictures, terminology and add explanations

### CODE REVIEW

**Review by itay1313 — APPROVED** [review]

File: pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md [hunk 1]
```diff
@@ -21,33 +21,40 @@ indicator: both
...
+* **Issuer Display Name** - The presentation name for the identity provider issuer
 
-* **OAuth 2.0 Token Endpoint** - An HTTPS URL that used in Code Flow
+* **JWT issuer (iss)** - The identifier of the principal that issued the JWT. The 'iss' value is a case-sensitive string containing a URL value.
 
-* **Choose IDP Certificate Type** - JWT Public Key / JWKS Endpoint
+* **Authentication Endpoint** - An HTTPS URL OAuth 2.0 Authorization Endpoint that is used by LivePerson to initiate the authentication flow with your idenitity principal. (this endpoint is invoked in case the conversation window opens as an external window)
 
-* **OAuth 2.0 Client ID** - Used in Code Flow
+* **Token Endpoint** - An HTTPS URL for token exchange. used in Code Flow.
```
Comment (lkisoslive) [inline pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md:47]: An HTTPS URL for token exchange, used in Code Flow.

Comment (lpbarber) [reply pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md]: Fixed

File: pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md [hunk 2]
```diff
@@ -21,33 +21,40 @@ indicator: both
...
 
 The following parameters should be defined:
 
 * **OAuth 2.0 Authentication Type** - Implicit / Code Flow
 
-* **OAuth 2.0 Authentication Endpoint** - An HTTPS URL that used in the case that the consumer conversation window is configured as a [Separate Browser Window](https://knowledge.liveperson.com/messaging-channels-web-messaging-authenticated-web-messaging.html#separate-browser-window-flow).
+* **Issuer Display Name** - The presentation name for the identity provider issuer
 
-* **OAuth 2.0 Token Endpoint** - An HTTPS URL that used in Code Flow
+* **JWT issuer (iss)** - The identifier of the principal that issued the JWT. The 'iss' value is a case-sensitive string containing a URL value.
```
Comment (j9t) [inline pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md:43]: Please use either double quotes (“”) or code styling (backticks) to mark “iss.”

File: pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md [hunk 3]
```diff
@@ -21,33 +21,40 @@ indicator: both
 
 ![Search for Consumer Identity Provider in the Management Console](img/mgmt_search_consumer_idp.png)
 
+![Consumer Identity Provider in the Management Console](img/mgmt_consumer_idp.png)
+
+
 {:start="3"}
 3.  Click on 'Click to configure'
```
Comment (j9t) [inline pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md:28]: I know, this issue existed before, but let’s fix the inconsistency and switch to double quotes here. Thanks!

File: pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md [hunk 4]
```diff
@@ -21,33 +21,40 @@ indicator: both
...
 * **OAuth 2.0 Authentication Type** - Implicit / Code Flow
 
-* **OAuth 2.0 Authentication Endpoint** - An HTTPS URL that used in the case that the consumer conversation window is configured as a [Separate Browser Window](https://knowledge.liveperson.com/messaging-channels-web-messaging-authenticated-web-messaging.html#separate-browser-window-flow).
+* **Issuer Display Name** - The presentation name for the identity provider issuer
 
-* **OAuth 2.0 Token Endpoint** - An HTTPS URL that used in Code Flow
+* **JWT issuer (iss)** - The identifier of the principal that issued the JWT. The 'iss' value is a case-sensitive string containing a URL value.
 
-* **Choose IDP Certificate Type** - JWT Public Key / JWKS Endpoint
+* **Authentication Endpoint** - An HTTPS URL OAuth 2.0 Authorization Endpoint that is used by LivePerson to initiate the authentication flow with your idenitity principal. (this endpoint is invoked in case the conversation window opens as an external window)
```
Comment (j9t) [inline pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md:45]: Either move the parentheses to within the sentence, or make it a separate sentence by starting with “This” and ending the sentence with a period.

File: pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md [hunk 5]
```diff
@@ -79,4 +86,4 @@ The brand's Authorization Service should have to register configuration of LiveP
 
 *	LP Popup redirect URI (when external web window is used)
```
Comment (j9t) [inline pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md:87]: Nit, but please use only a space between asterisk and text (i.e., “* Text”).

**Review by j9t — CHANGES_REQUESTED** [review]
Minor stuff—thanks for filing this.

File: pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md (hunk 5 above)
Comment (lpbarber) [reply pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md:87]: @j9t - I don't understand

Comment (j9t) [reply pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md:87]: For some reason these lists use a tab between asterisk and text of each list item entry—note, for example, the extraordinarily large space between “*” and “LP Popup”.

Screen Shot 2022-11-21 at 10 24 04

Please use a single space instead.

**Review by j9t — CHANGES_REQUESTED** [review]

**Review by j9t — APPROVED** [review]

**Pull request merged**